### PR TITLE
[stable/artifactory] Updating Artifactory version to 6.2.0

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory/templates/artifactory-statefulset.yaml [hunk 1]
```diff
@@ -75,19 +75,16 @@ spec:
                 cp -Lrf /bootstrap/* /artifactory_extra_conf/;
                 {{- end }}
                 if [ -d {{ .Values.artifactory.persistence.mountPath }}/access/etc/keys ]; then
```
Comment (rimusz) [inline stable/artifactory/templates/artifactory-statefulset.yaml]: remove:
if [ -d {{ .Values.artifactory.persistence.mountPath }}/access/etc/keys ]; then
fi;
which makes container crash

**Review by rimusz — COMMENTED** [review]
please address my comments

**Review by rimusz — APPROVED** [review]
/lgtm

**Comment by jainishshah17** [comment]
@mattfarina Can you give it `ok-to-test`.

**Comment by jainishshah17** [comment]
@rimusz ping

**Comment by jainishshah17** [comment]
Closing PR as chart is moved

**Pull request closed without merging**